tests/e2e/specs/recommended-widget.spec.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import {
+	activatePlugin,
+	loginUser,
+	visitAdminPage,
+} from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { waitForWpAdmin } from '../utils';
+
+describe( 'Recommended widget', () => {
+	jest.setTimeout( 30000 );
+	it( 'Widget should be available', async () => {
+		await loginUser();
+		await activatePlugin( 'wp-parsely' );
+		await visitAdminPage( '/widgets.php', '' );
+
+		await waitForWpAdmin();
+
+		await page.keyboard.press( 'Escape' );
+
+		await page.click( '.block-list-appender' );
+		await page.focus( '#block-editor-inserter__search-0' );
+		await page.keyboard.type( 'parse.ly recommended widget' );
```

Comment:
Similarly, could these be grouped into a new bit of code that is called `searchForParselyWidget` or similar? 

Could even then bundle that into a set of actions called `searchForAndAddParselyWidget` etc. Building up a list of human-describable actions, rather than programmatic steps, will make them more re-usable for new e2e tests, and move away from the declarative code of looking for specific classes on elements in the main bulk of tests.

Reply:
That does make sense. We will add more tests into this spec in the near future, for instance adding the widget and checking that it has some options, that it renders on the front-end, etc. It will be pretty useful to be able to reuse this actions.